DEVELOP.rst

```diff
@@ -29,11 +35,105 @@ set up your remote interpreter and test environment.
 Running the Tests
 =================
 
-You can run the tests like so::
+You can run the tests like::
+
+    vagrant ssh
+    cd /vagrant
+
+    # Run test suite
+    composer run test
+
+    # Run code style checks
+    composer run style
+
+    # Run code style checks
+    $ ./vendor/bin/phpcs ./src
+
+
+
+************
+Using Docker
+************
+
+Installation
+============
+
+Clone the project::
+
+    git clone git@github.com:crate/crate-dbal.git
+
+Setup environment::
+
+    composer install --prefer-source
```

Comment:
does this require to install `composer` + `php` + dependend libraries on the host maschine? I would not recommend doing so... I really would not want my local host system polluted with this. I think this should document how to run a 2nd php container with workspace mounted.